### PR TITLE
Rename project to PhoenixSlime

### LINKED ISSUE #9: Rename project?
PhoenixSlime ?

### PR DESCRIPTION
**Pull request by doomspork**
RFC @lpil — Feel free to merge if everything looks good to you.  Once it's merged I'll push the new hex package and add you to it.

Closes #9

### CODE REVIEW

**Comment by doomspork** [comment]
Also once this is merged and pushed I'll fixup the `phoenix_slime_example` app.

**Comment by lpil** [comment]
There's one missed reference to SlimFast in the README.

Is the extension still .slim? What's the intended alone file extension?
Do we support both for now?

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # Phoenix Template Engine for Slim
-[![Build Status](https://img.shields.io/travis/doomspork/phoenix_slim.svg)](https://travis-ci.org/doomspork/phoenix_slim) [![Hex Version](https://img.shields.io/hexpm/v/phoenix_slim.svg)](https://hex.pm/packages/phoenix_slim)
+[![Build Status](https://img.shields.io/travis/slime-lang/phoenix_slime.svg)](https://travis-ci.org/doomspork/phoenix_slim) [![Hex Version](https://img.shields.io/hexpm/v/phoenix_slim.svg)](https://hex.pm/packages/phoenix_slim)
 
-> Powered by [SlimFast](https://github.com/doomspork/slim_fast)
+> Powered by [SlimFast](https://github.com/slime-lang/slime)
```
Comment (doomspork) [inline README.md]: AH ha!

**Comment by doomspork** [comment]
Fixed up the `SlimFast` reference.  Good question RE: extension.  I think for backwards compatibility we should continue to support `.slim` but I'm open to others.   When you say "both", which other extension are you referring to?

**Comment by lpil** [comment]
I'm not sure. .slime ? That's what I assumed in the vim plugin.

I think we should have another extension so that editors can tell the difference and handle the embedded Ruby or Elixir code correctly. My editor occasionally gets confused by the Elixir with the .slim format.

**Comment by lpil** [comment]
I also think we should continue to support .slim

**Comment by doomspork** [comment]
I like `.slime`.  Any thoughts @henrik or @Rakoth?

**Comment by henrik** [comment]
Yeah, sounds good to me.

File: README.md [hunk 2]
```diff
@@ -25,11 +25,11 @@ config :my_app, MyApp.Endpoint,
     patterns: [
       ~r{priv/static/.*(js|css|png|jpeg|jpg|gif)$},
       ~r{web/views/.*(ex)$},
-      ~r{web/templates/.*(eex|slim)$}
+      ~r{web/templates/.*(eex|slime?)$}
```
Comment (doomspork) [inline README.md]: How's this @lpil?

Comment (henrik) [inline README.md]: No big deal, but I'd probably go with `eex|slim|slime` just for clarity.

Comment (doomspork) [inline README.md]: Good feedback, consider it done :+1:

Comment (lpil) [inline README.md]: :+1:

**Comment by lpil** [comment]
Looks good to go to me :)

File: README.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
 # Phoenix Template Engine for Slim
-[![Build Status](https://img.shields.io/travis/doomspork/phoenix_slim.svg)](https://travis-ci.org/doomspork/phoenix_slim) [![Hex Version](https://img.shields.io/hexpm/v/phoenix_slim.svg)](https://hex.pm/packages/phoenix_slim)
+[![Build Status](https://img.shields.io/travis/slime-lang/phoenix_slime.svg)](https://travis-ci.org/doomspork/phoenix_slim) [![Hex Version](https://img.shields.io/hexpm/v/phoenix_slim.svg)](https://hex.pm/packages/phoenix_slim)
```
Comment (doomspork) [inline README.md]: Missed a link.

**Pull request merged**